Split CLI install by OS, add Homebrew for Origin on Mac

adellape opened the pull request:
Per https://github.com/openshift/origin/issues/6689.

Pretty builds (internal):

http://file.rdu.redhat.com/~adellape/012216/homebrew_cli/cli_reference/get_started_cli.html#installing-the-cli

http://file.rdu.redhat.com/~adellape/012216/homebrew_cli/getting_started/administrators.html#downloading-the-binary

@smarterclayton PTAL.

smarterclayton commented:
The CLI download link disappears - it looks like brew is the only install path.  Maybe go ahead and have Windows, Mac, and Linux sections and describe each.  This is a fairly common question.

smarterclayton commented:
May want to say "Download the openshift-origin-clients-windows binary, unzip with a Zip program to a directory on your PATH, then run"

adellape commented:
Split the CLI install into Windows, Mac, and Linux steps. Also moved a Note box about requiring Git from `cli_ref/index` -> `cli_ref/get_started` as a Prerequisites section.

Origin build gets the Homebrew option for Mac:

http://file.rdu.redhat.com/~adellape/021516/origin/homebrew_cli/cli_reference/get_started_cli.html

Enterprise omits Homebrew and gets the RHSM option for RHEL:

http://file.rdu.redhat.com/~adellape/021516/enterprise/homebrew_cli/cli_reference/get_started_cli.html

@smarterclayton PTAL

tpoitras commented:
Thanks, @adellape 

One thing, I think this could this be explained more: "Then, unpack the archive to a directory on your PATH."

But I can address that in the videos.

adellape commented:
**revhistory note:**

(CLI Reference -> Get Started with the CLI)

```
Added a link:../cli_reference/get_started_cli.html#cli-prereqs[Prerequisites] section and added subsections to the link:../cli_reference/get_started_cli.html#installing-the-cli[Installing the CLI] section for link:../cli_reference/get_started_cli.html#cli-windows[Windows], link:../cli_reference/get_started_cli.html#cli-mac[Mac OS X], and link:../cli_reference/get_started_cli.html#cli-linux[Linux].
```
